### ISSUE TITLE
Custom loss function

### ISSUE THREAD

**Issue by Karlheinzniebuhr** [mljar/mljar-supervised#584]
I've implemented a custom loss function for a keras NN. I'd like to use it with Mljar, is there a way to plug it in?
Here is my keras example:

```
import keras.losses
import tensorflow as tf

# Create loss function
def sign_penalty(y_true, y_pred):
    penalty = 100.
    loss = tf.where(tf.less(y_true * y_pred, 0), \
                     penalty * tf.square(y_true - y_pred), \
                     tf.square(y_true - y_pred))

    return tf.reduce_mean(loss, axis=-1)

keras.losses.sign_penalty = sign_penalty  # enable use of loss with keras
print(keras.losses.sign_penalty)

# Create the model
model_2 = Sequential()
model_2.add(Dense(100, input_dim=scaled_train_features.shape[1], activation='relu'))
model_2.add(Dense(20, activation='relu'))
model_2.add(Dense(1, activation='linear'))

# Fit the model with our custom 'sign_penalty' loss function
model_2.compile(optimizer='adam', loss=sign_penalty)
history = model_2.fit(scaled_train_features, train_targets, epochs=25)

```

Any hint or instructions on how to integrate this would be highly appreciated

**Comment by pplonski** [comment]
Hi @Karlheinzniebuhr ,

You can defnie cusom eval metric in MLJAR AutoML. Please see details here https://github.com/mljar/mljar-supervised/issues/390#issuecomment-830049603

It is an evaluation metric used to decide on early stopping or model ensembling. It is not a loss function - it is not directly optimized. You can define the custom loss function in MLJAR AutoML.
